webpack.config.js

```diff
@@ -244,13 +207,4 @@ const config = {
 	},
 };
 
-switch ( process.env.NODE_ENV ) {
-	case 'production':
-		config.plugins.push( new webpack.optimize.UglifyJsPlugin() );
-		break;
-
-	default:
-		config.devtool = 'source-map';
```

Comment:
Did we intend to remove the `devtool` here? Currently investigating issues with sourcemaps, doesn't seem like this should have been impacted by the upgrade (aside from accommodating within new pattern for assigning environment).

Reply:
Yeah, I originally removed all "dev" related setup because while reading about WebPack, I thought the idea was that the "mode" was taking care of all these things automatically. Not the source map I guess